projects/ion/src/lib/chip/chip.component.scss

```
@@ -1,147 +1,119 @@
 @import '../../styles/index.scss';
+
+@mixin icon-icon($color) {
+  .icon-color {
+    ::ng-deep svg {
+      fill: $color;
+    }
+  }
+}
+
+@mixin icon-colors($background, $color) {
```

```suggestion
@mixin add-colors($background, $color) {
```
You should rename this to a generic name, since this mixin can be used for any element.